### ISSUE TITLE
[Bug Report] Extra render execution on VAutocomplete component

### ISSUE THREAD

**Issue by ZaidulinKirill** [vuetifyjs/vuetify#10838]
### Environment
**Vuetify Version:** 2.2.14
**Vue Version:** 2.6.11
**Browsers:** Chrome 80.0.3987.132
**OS:** Mac OS 10.15.3

### Steps to reproduce
1) Open console
2) There should be one "rendered" message.
3) Click on the autocomplete field. You will see the "No data available" message.
4) Check the console again. You will see a new "rendered" message

### Expected Behavior
The component should render only once because no reactive data or props are changed. 

### Actual Behavior
The component is rendered twice because the child "VAutocomplete" component trigger parent component to render

### Reproduction Link
<a href="https://codepen.io/zaidulinkirill/pen/yLNKQZx" target="_blank">https://codepen.io/zaidulinkirill/pen/yLNKQZx</a>

**Comment by johnleider** [comment]
This is resolved by performing this in nextTick:

https://github.com/vuetifyjs/vuetify/blob/02bdf288f666ecfd49cc04bdca7a814b09c7fb0f/packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts#L360

I don't particularly like that fix though.

**Comment by lioneaglesolutions** [comment]
This issue is being closed due to inactivity. If this is still present in the latest version, please submit a new issue with a reproduction.

If you have any additional questions, please reach out to us in our [[Discord community](https://community.vuetifyjs.com/)](https://community.vuetifyjs.com/).
